web/src/compose_ui.ts

```diff
@@ -624,6 +624,48 @@ export function handle_keyup(
     rtl.set_rtl_class_for_textarea($textarea);
 }
 
+/**
+ * True if the cursor in `$textarea` sits between an opening run of ` and its
+ * still‑missing matching closer.  Works for any run‑length (`, ```, …).
+ */
+export function cursor_inside_inline_span($textarea: JQuery<HTMLTextAreaElement>): boolean {
+    const textareaElement = $textarea[0];
+    if (!textareaElement || typeof textareaElement.selectionStart !== "number") {
```

Comment:
Why are you checking the type of `selectionStart` here? I assume one is expecting the textarea to be focused, with nothing selected, but it's not obvious to me how this check implements that.